packages/consumption/src/consumption/CoreErrors.ts

```diff
@@ -224,28 +224,28 @@ class Attributes {
     }
 
     public isNotSharedAttribute(attributeId: string | CoreId) {
-        return new CoreError("error.consumption.attributes.isNotSharedAttribute", `The attribute (id: ${attributeId}) is not a shared attribute.`);
+        return new CoreError("error.consumption.attributes.isNotSharedAttribute", `The Attribute (id: '${attributeId}') is not a shared Attribute.`);
     }
 
     public isNotOwnSharedAttribute(attributeId: string | CoreId) {
-        return new CoreError("error.consumption.attributes.isNotOwnSharedAttribute", `The attribute (id: ${attributeId}) is not an own shared attribute.`);
+        return new CoreError("error.consumption.attributes.isNotOwnSharedAttribute", `The Attribute (id: '${attributeId}') is not an own shared Attribute.`);
     }
 
     public isNotPeerSharedAttribute(attributeId: string | CoreId) {
-        return new CoreError("error.consumption.attributes.isNotPeerSharedAttribute", `The attribute (id: ${attributeId}) is not a peer shared attribute.`);
+        return new CoreError("error.consumption.attributes.isNotPeerSharedAttribute", `The Attribute (id: '${attributeId}') is not a peer shared Attribute.`);
     }
 
     public isNotThirdPartyOwnedRelationshipAttribute(attributeId: string | CoreId) {
         return new CoreError(
             "error.consumption.attributes.isNotThirdPartyOwnedRelationshipAttribute",
-            `The attribute (id: ${attributeId}) is not a third party owned RelationshipAttribute.`
+            `The Attribute (id: '${attributeId}') is not a third party owned RelationshipAttribute.`
         );
     }
 
     public senderIsNotPeerOfSharedAttribute(senderId: string | CoreAddress, attributeId: string | CoreId) {
         return new CoreError(
             "error.consumption.attributes.senderIsNotPeerOfSharedAttribute",
-            `The sender (id: ${senderId}) is not the peer you shared the attribute (id: ${attributeId}) with.`
+            `The sender (id: '${senderId}') is not the peer you shared the attribute (id: '${attributeId}') with.`
```

Comment:
- Is this the Sender of a Notification? If so, make this clear.
- "Attribute" instead of "attribute".